api/v1beta1/activemqartemis_types.go

```diff
@@ -69,6 +69,7 @@ type ActiveMQArtemisSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Ingress Domain",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	IngressDomain string `json:"ingressDomain,omitempty"`
 	// Specifies the template for various resources that the operator controls
+	//+kubebuilder:pruning:PreserveUnknownFields
```

Comment:
The new marker `//+kubebuilder:pruning:PreserveUnknownFields` for `ResourceTemplates` doesn't seem related to this PR.
```suggestion
```

Reply:
sorry I added it when debugging but forgot to delete it. will update.